tools/ClassGen/NodeGen.cpp

```diff
@@ -105,6 +105,8 @@ int main(int argc, char **argv) {
       .addInput("Input")
       .addInput("Weights")
       .addInput("Bias")
+      .addMember(MemberType::SizeT, "Axis")
```

Comment:
Have you seen nets where Axis is not 1?

Reply:
No. "Axis" uses default value 1 in Caffe2 zoo model.